### DISCUSSION TITLE
Universal copying

### DISCUSSION THREAD

**Discussion by FakeMoki** [Taewan-P/gpt_mobile#21] [Ideas]
I would like to see universal copying of responses from a chatbot. 
"Normal copying": by highlighting a certain text, it would be possible to copy it, rather than the entire answer. 
"Copy the code": with a click of a button, it would be possible to copy the code, rather than the entire answer.

**Comment by Taewan-P** [comment]
It is a great idea. However, this project is using a markdown renderer library to render the answers. I can't think of how I should implement that feature right now... Need further research.

**Reply by FakeMoki** [reply]
Fine. Then I will look forward to this feature in future updates 😊

**Comment by Taewan-P** [comment]
Highlighting certain text is now supported due to markdown library change. See #77 
It will be included in the next release.
Thanks
